x/gamm/pool-models/stableswap/pool_test.go

```diff
@@ -239,6 +239,47 @@ func TestScaledSortedPoolReserves(t *testing.T) {
 	}
 }
 
+func TestGetLiquidityIndexMap(t *testing.T) {
+	tests := map[string]struct {
+		poolAssets     sdk.Coins
+		scalingFactors []uint64
```

Comment:
Can we remove scaling factors here, and just pass in `[]uint64{}` to our constructor?

Reply:
done @ValarDragon